src/names/common.h

```diff
@@ -485,4 +485,16 @@ class CNameCache
 
 };
 
+struct NameNewReturn
+{
+    bool ok;
+    std::string err_msg;
+    std::string toaddress;
+    std::string hex;
+    std::string rand;
+    std::string data;
```

Comment:
I see.  I wrote my comment before I saw how you use the RPC calls, and think that this is fine.  Maybe add a comment explaining this to the struct?